backend/src/main/java/ai/verta/modeldb/utils/RdbmsUtils.java

```diff
@@ -2076,6 +2079,23 @@ public static VersioningEntry getVersioningEntryFromList(
     return versioningEntry.build();
   }
 
+  public static boolean checkConnectionsBasedOnPrivileges(
```

Comment:
can we move this in RoleService?
We should conditionally call this rather than have conditional execution within the function

Reply:
Moved function to RoleService and flag check on the individual method calls